src/StackExchange.Redis/ConnectionFailedEventArgs.cs

```diff
@@ -11,7 +11,19 @@ public sealed class ConnectionFailedEventArgs : EventArgs, ICompletable
     {
         private readonly EventHandler<ConnectionFailedEventArgs> handler;
         private readonly object sender;
-        internal ConnectionFailedEventArgs(EventHandler<ConnectionFailedEventArgs> handler, object sender, EndPoint endPoint, ConnectionType connectionType, ConnectionFailureType failureType, Exception exception, string physicalName)
+        private readonly string _physicalName;
+
+         /// <summary>
+         /// C'tor
+         /// </summary>
+         /// <param name="handler">Delegate to handle the event.</param>
+         /// <param name="sender">The source of the event.</param>
+         /// <param name="endPoint">Redis endpoint.</param>
+         /// <param name="connectionType">Redis connection type.</param>
+         /// <param name="failureType">Redis connection failure type.</param>
+         /// <param name="exception">The exception occured.</param>
+         /// <param name="physicalName">Connection physical name.</param>
+        public ConnectionFailedEventArgs(EventHandler<ConnectionFailedEventArgs> handler, object sender, EndPoint endPoint, ConnectionType connectionType, ConnectionFailureType failureType, Exception exception, string physicalName)
```

Comment:
similar to the comment on `EndPointEventArgs` - I wonder if the `public` .ctor should not expose `handler` / `sender`

Reply:
Answer is the same.